RecoEgamma/ElectronIdentification/python/Identification/heepElectronID_HEEPV71_cff.py

```diff
@@ -0,0 +1,95 @@
+import FWCore.ParameterSet.Config as cms
+from PhysicsTools.SelectorUtils.centralIDRegistry import central_id_registry
+
+# Common functions and classes for ID definition are imported here:
+from RecoEgamma.ElectronIdentification.Identification.heepElectronID_tools import HEEP_WorkingPoint_V1,configureHEEPElectronID_V71
+
+#the same HEEP V70 but now has calo et cut of >5 GeV and isolation cut is relaxed vs calo et
+
+# The cut values for the  Barrel and Endcap
+idName = "heepElectronID-HEEPV71"
+WP_HEEP71_EB = HEEP_WorkingPoint_V1(
```

Comment:
why is this not just ```WP_HEEP71_EB = WP_HEEP70_EB.clone(idName)``` ?
copy-paste is not the best development strategy.

Reply:
and here

Reply:
agreed, done